extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzExtensionPointConfig.java

```diff
@@ -20,4 +21,31 @@
     @ConfigItem
     @ConfigDocMapKey("property-name")
     public Map<String, String> properties;
+
+    @ConfigGroup
+    static public class QuartzExtensionPointOptionalConfig {
+        /**
+         * Class name for the configuration.
```

Comment:
This is the JobStore class name. I think we could update the documentation to read as such. 
Any additional information like - the class has to implement JobStore interface will be very useful to the user.

Reply:
I think that the point is to have a reusable config group, i.e. an alternative to `QuartzExtensionPointConfig`. In other words, the additional info should go to the `QuartzBuildTimeConfig#customStore`.

Reply:
Revise to add comments to field customStore , "For instance,..."

```java
@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
public class QuartzBuildTimeConfig {
...
    /**
     * Custom job store.
     * <p>
     * For instance,
     * <ul>
     * <li>quarkus.quartz.custom-store.class=org.acme.quartz.MyCustomJobStore
     * <li>quarkus.quartz.custom-store.properties.myProp1=MyPropValue1
     * <li>quarkus.quartz.custom-store.properties.myProp2=MyPropValue2
     * </ul>
     */
    @ConfigItem
    @ConfigDocMapKey("custom-store")
    @ConfigDocSection
    protected QuartzExtensionPointOptionalConfig customStore;
...
```